eland/tests/dataframe/test_groupby_pytest.py

```diff
@@ -154,3 +154,17 @@ def test_groupby_different_dtypes(self, by):
     def test_groupby_dropna(self):
         # TODO Add tests once dropna is implemeted
         pass
+
+    def test_groupByDataFrame_count(self):

```

Comment:
nit: use snake case not camelcase: `test_groupby_dataframe_count()`

Reply:
Oops 😐